### PR TITLE
Update emitter docs to include configuration examples

### PR DESCRIPTION
**Pull request by amsoedal**
For a TypeSpec beginner, it may be difficult to understand how to configure or format tspconfig.yaml to tweak the different options available. Adding a basic example should help folks get started.

### CODE REVIEW

File: docs/emitters/json-schema/reference/emitter.md [hunk 1]
```diff
@@ -21,6 +21,15 @@ emit:
   - "@typespec/json-schema"
 ```
 
+The config can be extended with options as follows:
+
+```yaml
+emit:
+  - "@typespec/json-schema"
+options:
+  "@typespec/json-schema": option:value
```
Comment (timotheeguerin) [inline docs/emitters/json-schema/reference/emitter.md:30]: I think it's missing a new line and indentation here

Comment (amsoedal) [reply docs/emitters/json-schema/reference/emitter.md:30]: I'm struggling to get it formatted correctly -- any suggestions? This is how it looks in code, which *should* in theory be right

image

Comment (amsoedal) [reply docs/emitters/json-schema/reference/emitter.md]: Thanks to Timothee for the fix! Was missing a space between "option:" and "value"

**Review by timotheeguerin — APPROVED** [review]
Nice thanks

**Pull request merged**